<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fight</class>
 <widget class="QDialog" name="Fight">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1088</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QFrame" name="mid_blackline">
   <property name="geometry">
    <rect>
     <x>543</x>
     <y>310</y>
     <width>2</width>
     <height>71</height>
    </rect>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="under_blackline">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>380</y>
     <width>1101</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="over_blackline">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>310</y>
     <width>1101</width>
     <height>2</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: black;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="colour_back">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>310</y>
     <width>1111</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: gray;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QLabel" name="Hero_HP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>:)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Enemy_HP">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>:)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Hero_Mana">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>10</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>:)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Enemy_Mana">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>:)</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="spellListWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>541</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="combatLogTextEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>80</y>
      <width>511</width>
      <height>121</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_escape">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>220</y>
      <width>111</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Втеча</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="enemyImageLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1091</width>
     <height>301</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::LeftToRight</enum>
   </property>
   <property name="text">
    <string>:)</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <zorder>colour_back</zorder>
  <zorder>under_blackline</zorder>
  <zorder>mid_blackline</zorder>
  <zorder>over_blackline</zorder>
  <zorder>enemyImageLabel</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
